### ISSUE TITLE
Example from demo does not work

### ISSUE THREAD

**Issue by itelmenko** [adeptoas/sweet-modal-vue#19]
Hello!

I have tried this example:

```
<sweet-modal icon="error" hide-close-button blocking overlay-theme="dark" modal-theme="dark">
	This is an error ...

	<sweet-button slot="button" color="red" v-on:click="closeExample('darkWithBlockingError')">Press this Button</sweet-button>
</sweet-modal>
```
and I get error

> [Vue warn]: Unknown custom element: <sweet-button> - did you register the component correctly? For recursive components, make sure to provide the "name" option.

Also if I use example:

```
<sweet-modal>This is an alert.</sweet-modal>
```

I have not errors. But I see empty white layer without dialog window after `this.$refs.modal.open();`
Div with class `.sweet-modal` has style `display:none;`

I have tried on Vue 2.1.10 and Vue 2.3.3

**Comment by bluefirex** [comment]
For <sweet-button> as stated in the README you have to supply your own implementation. SweetModal does not come with buttons.

As for the other issue I'll look into it.

**Comment by mirzap** [comment]
I have the same issue. It's not related to this sweet-button. As @itelmenko stated even most simple example `<sweet-modal>This is an alert.</sweet-modal>` does not work. It just shows an empty white or black layer (depending on theme). 

And if I change style for `.sweet-modal` class to `display:block` modal appears.

**Comment by mirzap** [comment]
I found issue in my case. I'm using Laravel Spark in current project, and by default sweet-alert css is loaded which contains some collision classes. Removed that in works like a charm. 
@itelmenko you probably have some collision class to.

**Comment by jonnywilliamson** [comment]
@mirzap Ah yes, just discovered this as well.

Having sweet-alert installed causes lots of css problems. What a shame.

**Comment by mikerogne** [comment]
@mirzap Any guidance on a fix, or what specific classes are conflicting?

**Comment by mikerogne** [comment]
Ok, so... within Laravel Spark application, public/css/sweetalert.css does have something like:

`.sweet-alert { ... display: none; ... }`

That's... weird? Changing to block does fix it for me.

**Comment by bluefirex** [comment]
I pushed a possible fix for this issue in the master branch (f8b3ac35c1cb26c994ccecc6523638a864f28547). Can you try if that solves it?

**Comment by mikerogne** [comment]
Not sure how I test it?

**Comment by bluefirex** [comment]
Remove the current version from npm, clone this repo, run `npm i` on it and manually place it in the node_modules folder

**Comment by mikerogne** [comment]
Got it, using: `"sweet-modal-vue": "adeptoas/sweet-modal-vue#master",` in package.json

Shows: `└── sweet-modal-vue@1.2.0  (git://github.com/adeptoas/sweet-modal-vue.git#f8b3ac35c1cb26c994ccecc6523638a864f28547)`

**Comment by mikerogne** [comment]
And actually, that both fixes the display issue itself, AND the non-centered issue I was having :)

**Comment by bluefirex** [comment]
Awesome! I'll release this fix in an update in a few days together with a few other fixes.

**Comment by mikerogne** [comment]
Fantastic, thanks a bunch!!! 👍  👍

**Comment by itelmenko** [comment]
It does not work for me still.
Because:
```
[Vue warn]: Unknown custom element: <sweet-button> - did you register the component correctly? For recursive components, make sure to provide the "name" option.
```
What is sweet-button?

**Comment by itelmenko** [comment]
However  if I don't use sweet-button It also does not work for me.

**Comment by bluefirex** [comment]
`sweet-button` is only available in the demo.

What does not work for your?

**Comment by itelmenko** [comment]
> What does not work for your?
All details in start post is actual after removing sweet-button also

**Comment by bluefirex** [comment]
Fixed in c05fb535b10cb39d2bfa17c218d198471a1e37da + f8b3ac35c1cb26c994ccecc6523638a864f28547, released in [v2.0.0](https://github.com/adeptoas/sweet-modal-vue/releases/tag/v2.0.0)
